soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4r5xx

```diff
@@ -11,4 +11,8 @@ config SOC
 config NUM_IRQS
 	default 95
 
+config DMAMUX_STM32
```

Comment:
Already selected in https://github.com/zephyrproject-rtos/zephyr/blob/47ebde30b9ea0efded1faf7920bb3a72f0d3664a/drivers/dma/Kconfig.stm32#L28